SIPs/SIP-000X.md

```diff
@@ -0,0 +1,161 @@
+
+**Issuance**  
+Proposals made to the Issuance track SHOULD be used to increase the total supply of SOV tokens.
+
+* All code, documentation, and content funded by an Issuance track proposal SHOULD be released under one of the following appropriate licenses:  
+  * Creative Commons Zero (CC-0)  
+  * MIT  
+
+**Meta**  
+Proposals made to the Meta track SHOULD affect changes to this document. SIP Editors have the authority to fix errata in this document on an as-needed basis without going through the Sovryn governance process. All other proposals to modify this document SHOULD be made to the Meta track.
+
+**Proclamations**  
+Proposals made to the Proclamations track SHOULD be used to make a statement on behalf of Sovryn and can say whatever the author(s) want, so long as the proclamation is consistent with the Sovryn Constitution.
+
+* For example, a Proclamation can be used to signal the Sovryn community's support for or disapproval of a certain RSKIP.  
+
+### Stage II: Pre-proposal
+During Stage II you SHOULD seek feedback on your SIP idea by sharing it with your peers in the Sovryn community and soliciting their feedback. Examples of appropriate venues to share your SIP idea include:
+
+* The [#bitocracy channel](https://discord.gg/WApXyz4D5h) in the Sovryn Discord  
+* The [SIP category](https://forum.sovryn.app/c/bitocracy/sip-sovryn-improvement-proposals/) in the Sovryn forum  
+* The Issues section of the SIPs repo  
+
+Be open-minded and respectful of all feedback you receive. Adjust your proposal to address legitimate concerns as they come up to increase the odds of your proposal passing review in later stages.
+
+### Stage III: Draft
+After you have asked the Sovryn community whether an idea has any chance of support, and you have received sufficient feedback to feel confident going forward, you SHOULD create a draft SIP as a draft pull request to the SIPs repo. Use a template from the Templates section below to ensure you are including all the necessary information. Draft SIP files submitted to GitHub SHOULD be located in the `SIPs` directory and given a temporary name e.g. `SIP.md`, which the SIP Editor will later assign an SIP number, and SHOULD comply with the requirements set forth below to maintain consistency between SIPs.
+
+* A SIP MAY be drafted and submitted from anywhere. The SIPs repo is used only as a matter of convenience and to ease coordination between Sovryn contributors.
+
+**Templates**  
+Below is a list of SIP templates for each track. Copy the template for the track your SIP is in, fill it out, and submit the pull request with your SIP for review. Sections marked as “required” in the template MUST be completed. Note that all SIPs MUST be licensed CC-0.
+
+* [Contract](../templates/contract_template.md)  
+* [Finance](../templates/finance_template.md)  
+* [Issuance](../templates/issuance_template.md)  
+* [Meta](../templates/meta_template.md)  
+* [Proclamations](../templates/proclamation_template.md)  
```

Comment:
these links are not working here

Reply:
@ororopickpocket yes, because I moved the SIP PR here: https://github.com/DistributedCollective/SIPS/pull/11 and the template files here: https://github.com/DistributedCollective/SIPS/pull/10